docs/pages/components/Notice.mdx

```diff
@@ -0,0 +1,192 @@
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      Caution
+    </h3>
+  }
+>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+### Notice With Different icon
+
+<Note>
+  The `icon` prop takes a string that corresponds to a [Cauldron
+  Icon](/components/icon). If you would like to use a custom icon, you can
+  compose the contents of the `Notice` component (see
+  <a href="/components/Notice#notice-with-composed-contents">example</a>).
+</Note>
+
+```jsx example
+<Notice type="caution" title="Dynamo!" icon="bolt">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+### Notice With Composed Contents
+
+```jsx example
+<Notice>
+  <div className="Notice__title">
+    <Icon type="star" />
+    <h2
+      style={{
+        margin: 0,
+        padding: 0,
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      You are a star!
+    </h2>
+  </div>
+  <p style={{ margin: 'var(--space-half) 0 0 var(--space-half)' }}>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at
+    porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus
+    arcu condimentum sed.
+  </p>
+</Notice>
+```
+
+### Notice (thin)
+
+Rendering a `Notice` that appears as a thin bar can be accomplished by only passing a `title` prop, or only passing a child.
+
+```jsx example
+<Notice
+  type="caution"
+  title="Caution, about to tread into dangerous territory"
+/>
+
+<Notice type="info">
+  <div
+    className="Notice__content"
+    style={{ display: 'inline-flex', gap: 'var(--space-half)', justifyContent: 'center', alignItems: 'center' }}
+  >
+  <Icon type="info-circle-alt" />
+    <span>A new update has been released!</span>
+  </div>
+</Notice>
+
+```
+
+### Notice (with HTML element properties)
+
+The `Notice` component will support the spreading of any props normally available to a `div` element. This includes
+properties such as `role`, `aria-label`, `aria-labelledby`, `id`, etc.
+
+```jsx example
+<Notice type="info" title="Alert!" role="alert">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+## Props
+
+<ComponentProps
+  children={{ required: false }}
+  props={[
+    {
+      name: 'type',
+      type: ["'info'", "'caution'"],
+      defaultValue: '"info"',
+      description:
+        'Notice preset variant that sets both the background color and Icon type (optionally overwritable)',
+      required: false
+    },
+    {
+      name: 'title',
+      type: 'React.ReactNode',
```

Comment:
This should be [`ContentNode`](https://github.com/dequelabs/cauldron/blob/82de49c0307ba11266fe348884852b039e3ead9b/packages/react/src/types.ts#L10-L19) as we specifically use that type to avoid things like `boolean` or `null`.